src/main/webapp/WEB-INF/views/register-user.jsp

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Register User</title>
+<link rel="stylesheet" type="text/css"
+	href="css/registrationPageStyle.css">
+<link rel="stylesheet"
+	href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://use.typekit.net/lot0wjb.css">
+</head>
+<body>
+	<header>
+
+		<!-- USER ICON-->
+		<i class="fa fa-user-circle-o"
+			style="float: right; font-size: 56px; color: #554D41"></i>
+	</header>
+
+	<form action="/register" method="post" id="registrationForm">
+	
+		<h1>Register User</h1>
+		
+			<label for="firstName">First Name</label> 
+			<input type="text" id="firstName" name="firstName" required="required"><br>
+		    <label for="lastName">Last Name</label> 
+		    <input type="text" id="lastName" name="lastName" required="required"><br> 
+			<label for="emailAddress">Email</label> 
+			<input type="text" id="emailAddress" name="emailAddress" required="required"><br> 
+			<label for="phoneNumber">Phone Number</label> 
+			<input type="text" id="phoneNumber" name="phoneNumber" required="required"><br>
+			<label for="password">Password</label> 
+			<input type="password" id="password" name="password" required="required" oninput="validatePasswordRequirements()"><br> 
+            <span id="passwordError" style="color: red;"></span><br> <!-- Error message for password -->
+            <label for="confirmPassword">Confirm Password</label> 
+            <input type="password" id="confirmPassword" name="confirmPassword" required="required" oninput="validatePassword()"><br> 
+            <span id="confirmPasswordError" style="color: red;"></span><br> <!-- Error message for confirm password -->
+			<button type="button" onclick="clearFields()" >Cancel</button>
+			<button type="submit" id="submitButton" class="disabled-button">Submit</button>
+	</form>
+
+	<script>
+		// Check if passwords requirements are met
+		function validatePasswordRequirements() {
+			var password = document.getElementById('password').value;
+			var passwordError = document.getElementById('passwordError');
+
+			var regex = /^(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*]).{6,}$/;
+
+			if (!regex.test(password)) {
+				passwordError.textContent = 'Password must have at least 6 characters, 1 uppercase letter, 1 number, and 1 special character (!@#$%^&*).';
```

Comment:
We need min. of 8 characters for the password according to the assignment brief.